pool/_RPC_dry.gno

```diff
@@ -58,9 +58,12 @@ func DrySwap(
 		feeGrowthGlobalX128 = pool.feeGrowthGlobal1X128
 	}
 
-	pool.slot0.unlocked = false
+	// pool.slot0.unlocked = false
```

Comment:
Is this dead code ??? 
If yes, plz remove it

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/403/commits/5b0b453529c9b3d1957752637f5e0aacb4ed4cbb